projects/js-packages/connection/composer.json

```
@@ -0,0 +1,4 @@
+{
+	"name": "automattic/js-connection",
+	"description": "Jetpack Connection Component."
+}
```

I recommend adding changelogger right away. And you'll want your tests to be run by the CI infrastructure.
```suggestion
	"require": {},
	"require-dev": {
		"automattic/jetpack-changelogger": "1.1.x-dev"
	},
	"scripts": {
		"test-js": [
			"Composer\\Config::disableProcessTimeout",
			"yarn install",
			"yarn test"
		],
		"test-coverage": [
			"Composer\\Config::disableProcessTimeout",
			"yarn install",
			"whatever command will run the tests and produce coverage at $COVERAGE_DIR/clover.xml"
		]
	},
	"repositories": [
		{
			"type": "path",
			"url": "../../packages/*",
			"options": {
				"monorepo": true
			}
		}
	]
}
```
If you'll need a build step, that would go inside `scripts` too, like
```
		"build-development": [
			"Composer\\Config::disableProcessTimeout",
			"yarn build"
		]

```
You can have a "build-production" script too if there's a difference.

Once you've decided on a mirror repo, you'd also want to add
```
	"extra": {
		"mirror-repo": "Automattic/????",
		"changelogger": {
			"link-template": "https://github.com/Automattic/????/compare/v${old}...v${new}"
		}
	}
```
in there, with the `????` replaced of course.